### DISCUSSION TITLE
DeepXDE for shocktube problem: non continuous pressure density

### DISCUSSION THREAD

**Discussion by misaacs** [lululxvi/deepxde#1172] [Q&A]
How would you use deepXDE to model compressible noncontinuous fluids? Any examples or suggestions? This would involve velocity, density and pressure... For instance,  the sod shocktube problem : https://en.wikipedia.org/wiki/Sod_shock_tube#%3A%7E%3Atext%3DThe_Sod_shock_tube_problem%2Cstates_of_an_ideal_gas?wprov=sfla1

**Comment by lululxvi** [comment]
Check this https://www.sciencedirect.com/science/article/pii/S0045782519306814

**Reply by tangri** [reply]
Is there a code or a repository associated with this?
